content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md

```diff
@@ -96,7 +101,90 @@ Once an override has been registered, it will exist until explicitly removed. To
+        }
+    }
+}
+```
+
+{{< note >}}
+If you manually enable or disable the DPE through the Console Variables editor, you might need to close and reopen the Entity Inspector.
+{{< /note >}}
+
+Second, to enable prefab overrides in inspector, you can create a settings registry file called `editorpreferences.setreg` with the following contents:
+
+```json
+{
+    "O3DE": {
+        "Preferences": {
+            "Prefabs": {
+                "EnableInspectorOverrideManagement": true
+            }
+        }
+    }
+}
+```
+
+{{< note >}}
+You can reuse the existing settings registry file `editorpreferences.setreg` if you have already created.
+{{< /note >}}
+
+### Override visualization
+
+When you make an override edit in the inspector, you will notice that a blue circle appears before the property label and the label text is bolded. In the component header, a blue circle is also shown near the text, which indicates that an owning property value has been overridden.
+
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-component-property.png" width="500" alt="Overriding a component property." >}}
+
+If a component is added as an override, a different blue circle with a plus sign appears instead.
+
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-add-component-override.png" width="500" alt="Adding a component as an override." >}}
+
+### Revert property overrides
+
+With the inspector override support, you can now revert overridden property values respectively. Unlike reverting overrides on an entity, you can selectively revert the property you want:
+
+1. In Entity Inspector, **right-click** on the property label or the blue circle and choose **Revert override** from the context menu.
+1. Notice that the property no longer has any indication of having an override.
+
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-revert-component-property-override.png" width="500" alt="Reverting a component property override." >}}
+
+{{< note >}}
+Once a property value is overridden to non-default value, the override edit can only be removed by doing a revert. In other words, setting the value back to default does not remove the override edit.
+{{< /note >}}
+
+In addition, you are able to revert all property overrides that are owned by a component:
+
+1. In Entity Inspector, **right-click** on the blue circle in the component header and choose **Revert Overrides** from the context menu.
+1. Notice that all the properties no longer have any indication of having an override.
+
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-revert-component-override.png" width="500" alt="Reverting a component added as an override." >}}
+
+For a component added as an override, it can be reverted in the same way mentioned above. Alternatively, you can delete the component with either of the following methods:
```

Comment:
Shouldn't we be also talking about the case where a component is removed as an override and how to revert it?

Reply:
Good callout. I will mention that it can be reverted by doing an undo or a 'revert' on the entity. It should be mentioned for an entity deleted as an override as well. I will add both.